integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/addsegment/AddSegmentTestCase.scala

```diff
@@ -962,6 +962,88 @@ class AddSegmentTestCase extends QueryTest with BeforeAndAfterAll {
     sql(s"drop table $tableName")
   }
 
```

Comment:
I don't understand why these test cases cannot be added in the testCleanFilesCommand cases. You can any necessary function that you want to use there. Also please merge these 2 test cases into 1. And also Include Inprogress segment also.

Reply:
And also add similar test case with partition table too.

Reply:
TestCases Added for IN_PROGRESS segments also